src/client/src/ui/spotTile/components/analyticsTile/AnalyticsTileChart.tsx

```diff
@@ -1,40 +1,44 @@
-import React from 'react'
-import { LineChart, ResponsiveContainer, Line, YAxis, ReferenceLine } from 'recharts'
+import React, { FC } from 'react'
+import { LineChart, ResponsiveContainer, Line, YAxis, ReferenceLine, Tooltip } from 'recharts'
 import { styled } from 'rt-theme'
-import { seriesData } from '../test-resources/spotTileProps'
+import { SpotPriceTick } from '../../model'
+import AnalyticsTileTooltip from './AnalyticsTileToolTip'
 
 const AnalyticsTileChartWrapper = styled.div`
   width: 100%;
   height: 100%;
 `
 
-class AnalyticsTileChart extends React.Component {
-  render() {
-    const lineProps = { strokeDasharray: '4 3', stroke: '#737987', strokeOpacity: 0.9, strokeWidth: 0.8 }
-    return (
-      <AnalyticsTileChartWrapper>
-        <ResponsiveContainer width="100%" height="100%">
-          <LineChart data={seriesData} margin={{ top: 0, right: 0, bottom: 0, left: 0 }}>
-            <defs>
-              <linearGradient id="lineColour" x1="0" y1="0" x2="0" y2="1">
-                <stop stopColor="#737987" stopOpacity={1} strokeWidth={1.2} />
-              </linearGradient>
-            </defs>
-            <Line dataKey="y" dot={false} stroke="url(#lineColour)" />
-            <YAxis
-              width={0}
-              domain={['dataMin', 'dataMax']}
-              axisLine={false}
-              tickLine={false}
-              padding={{ top: 0, bottom: 0 }}
-              tick={false}
-            />
-            <ReferenceLine y={0} {...lineProps} />
-          </LineChart>
-        </ResponsiveContainer>
-      </AnalyticsTileChartWrapper>
-    )
-  }
+interface Props {
+  history: SpotPriceTick[]
+}
+const AnalyticsTileChart: FC<Props> = ({ history }) => {
```

Comment:
In some places throughout the project we are doing:
```typescript
import React from 'react'
const SomeComponent: React.FC<Props> ...
```
whereas in others
```typescript
import React, { FC } from 'react'
const SomeComponent: FC<Props> ...
```
The same applies to class components (`extends React.Component` VS `extends Component`)

It's not a huge deal but I think we should consistently follow one - I'd vote for the first one as it's currently the approach used in most places

Reply:
I like the latter